tools/scripts/build-rust-android.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Let's make the script fail upon any command exiting with a non-zero status code.

```suggestion
#! /bin/bash
set -e
```